cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/distributed/DistributedMapReduceTaskContextProvider.java

```diff
@@ -126,31 +105,6 @@ protected void shutDown() throws Exception {
   }
 
   private static Injector createInjector(CConfiguration cConf, Configuration hConf) {
-    return Guice.createInjector(
```

Comment:
there seem to be some differences between these bindings and the ones in ``AbstractProgramTwillRunnable``. Ok to refactor them as ``DistributedProgramRunnableModule``?